Makefile

```diff
@@ -120,6 +120,23 @@ pkg/openapi/client.go: bin/openapi-generator-cli api/openapi/model-registry.yaml
 		--ignore-file-override ./.openapi-generator-ignore --additional-properties=isGoSubmodule=true,enumClassPrefix=true,useOneOfDiscriminatorLookup=true
 	gofmt -w pkg/openapi
 
+# Start the MySQL database
+.PHONY: start/mysql
+start/mysql:
```

Comment:
@Al-Pragliola will we need CI tests per DB schema DDL?

Reply:
Maybe parameterize the script to start different DB types from Dockerfile? I wonder if there are ready made gha for this. @dbasunag , @fege or @lugi0 do you know?